server/config/config.go

```diff
@@ -740,6 +740,15 @@ type ScheduleConfig struct {
 
 	// The day of hot regions data to be reserved. 0 means close.
 	HotRegionsReservedDays int64 `toml:"hot-regions-reserved-days" json:"hot-regions-reserved-days"`
+
+	// OperatorTimeFactor is the time factor for operator.
+	// The max duration of one operator step is: multi(region_size, factor)
+	// Default: 6 s/MB
+	OperatorTimeFactor uint64 `toml:"operator-time-factor" json:"operator-time-factor"`
+
+	// MaxRegionSize is the max size of region.
+	// default: 96MB
+	MaxRegionSize uint64 `toml:"max-region-size" json:"max-region-size"`
```

Comment:
What if we change it by API, is it expected?

Reply:
I think it can be changed by config file.